src/Sulu/Bundle/MediaBundle/Controller/FormatController.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MediaBundle\Controller;
+
+use FOS\RestBundle\Routing\ClassResourceInterface;
+use Sulu\Bundle\MediaBundle\Media\FormatManager\FormatManagerInterface;
+use Sulu\Bundle\MediaBundle\Media\FormatOptions\FormatOptionsManagerInterface;
+use Sulu\Component\Rest\RestController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Makes the image formats, with the format options available through the REST API.
+ */
+class FormatController extends RestController implements ClassResourceInterface
+{
+    /**
+     * Returns all format resources.
+     *
+     * @param $id
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function cgetAction($id, Request $request)
+    {
+        $formatOptions = $this->getFormatOptionsManager()->getAll($id);
+        $formats = $this->getFormatManager()->getFormatDefinitions($this->getLocale($request), $formatOptions);
+
+        return $this->handleView($this->view($formats));
+    }
+
+    /**
+     * Edits a format resource.
+     *
+     * @param int $id
+     * @param string $key
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function putAction($id, $key, Request $request)
+    {
+        $options = $request->get('options', []);
+
+        if (empty($options)) {
+            $this->getFormatOptionsManager()->delete($id, $key);
+        } else {
+            $this->getFormatOptionsManager()->save($id, $key, $options);
+        }
+
+        $formatOptions = $this->getFormatOptionsManager()->get($id, $key);
```

Comment:
Note: It was not possible for the FormatManager to load the formatOptions on its own, due to a circular reference (MediaManager -> FormatManager- > FormatOptionsManager -> MediaManager). After implementing it I like the current solution more, as its possible to load the image-formats without any media-specific options.